Dockerfile.assisted-installer-deployment

```diff
@@ -30,3 +26,8 @@ WORKDIR /home/assisted-installer-deployment
 COPY . .
 
 RUN pip install .
+
+# required for generating configuration/bundle in assisted-service repo
+FROM registry.ci.openshift.org/openshift/release:golang-1.16
+COPY ./setup_env.sh ./
+RUN ./setup_env.sh assisted_service
```

Comment:
this is probably problematic, as it's probably changes the current directory. no?

Reply:
why we need to be based on ``registry.ci.openshift.org/openshift/release:golang-1.16``?

Reply:
no mandatory, added to dnf install instead